numpy/ctypeslib.py

```diff
@@ -569,10 +570,9 @@ def as_array(obj, shape=None):
 
     def as_ctypes(obj):
         """
-            as_ctypes(obj)
-
-            Create and return a ctypes object from a numpy array.  Actually
-            anything that exposes the __array_interface__ is accepted.
+            Create and return a ctypes array from a numpy array.  Actually
```

Comment:
```suggestion
            Create and return a ctypes array from a numpy array.
```
The removed adverb should be used rarely and with extreme caution.